src/modules/dashboard/components/ActionsPage/TotalStakeWidget/TotalStakeWidget.tsx

```diff
@@ -0,0 +1,154 @@
+    defaultMessage: '{totalPercentage}% of {requiredStake} {tokenSymbol}',
+  },
+  userStake: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.userStake',
+    defaultMessage: `You staked {userPercentage}% of this motion ({userStake} {tokenSymbol}).`,
+  },
+});
+
+const TotalStakeWidget = ({
+  colonyAddress,
+  motionId,
+  isObjectionStake = false,
+}: Props) => {
+  const { walletAddress } = useLoggedInUser();
+  const { data, loading } = useStakeAmountsForMotionQuery({
+    variables: {
+      colonyAddress,
+      userAddress: walletAddress,
+      motionId,
+      isObjectionStake,
+    },
+  });
+  const {
+    data: nativeTokenAddressData,
+    loading: loadingNativeTokenAddress,
+  } = useColonyNativeTokenQuery({
+    variables: { address: colonyAddress },
+  });
+  const [
+    fetchTokenInfo,
+    { data: tokenInfoData, loading: loadingTokenInfoData },
+  ] = useTokenInfoLazyQuery();
+
+  useEffect(() => {
+    if (nativeTokenAddressData) {
+      const {
+        processedColony: { nativeTokenAddress },
+      } = nativeTokenAddressData;
+      fetchTokenInfo({ variables: { address: nativeTokenAddress } });
+    }
+  }, [fetchTokenInfo, nativeTokenAddressData]);
+
+  if (loading || !data?.stakeAmountsForMotion) {
+    return null;
+  }
+
+  const { totalStaked, userStake, requiredStake } = data.stakeAmountsForMotion;
+  const divisibleRequiredStake = requiredStake !== '0' ? requiredStake : 1;
+  const totalStakedPercentage = bigNumberify(totalStaked)
+    .mul(100)
+    .div(divisibleRequiredStake)
+    .toNumber();
+  const userStakePercentage = bigNumberify(userStake)
+    .mul(100)
+    .div(divisibleRequiredStake)
+    .toNumber();
+  const formattedTotalStakedPercentage = formatNumber({
+    truncate: 2,
+  })(totalStakedPercentage);
+  const formattedUserStakePercentage = formatNumber({
+    truncate: 2,
+  })(userStakePercentage);
+
+  return (
+    <div className={styles.widget}>
+      <div className={styles.widgetHeading}>
+        <Heading
+          appearance={{
+            theme: 'dark',
+            size: 'small',
+            weight: 'bold',
+            margin: 'none',
+          }}
+          text={isObjectionStake ? MSG.objectionTitle : MSG.motionTitle}
+          className={styles.title}
+        />
+        <span className={styles.stakeProgress}>
+          {!loadingTokenInfoData && !loadingNativeTokenAddress && (
+            <FormattedMessage
+              {...MSG.stakeProgress}
+              values={{
+                totalPercentage: formattedTotalStakedPercentage,
+                requiredStake,
+                tokenSymbol: tokenInfoData?.tokenInfo.symbol,
+              }}
+            />
+          )}
+        </span>
+      </div>
+      <ProgressBar
+        value={totalStakedPercentage}
+        max={100}
+        appearance={{
+          barTheme: isObjectionStake ? 'danger' : 'primary',
+          backgroundTheme: 'default',
+        }}
+      />
```

Comment:
So apparently this is not correct for a spec perspective.

I've talked to Product today about this, so here's the main points:
- If we only have staking for YES, then we just show the Progress bar for YES
- If we have stake for both YES and NO then we show a different UI _(the one with the horizontal radio buttons)_
- If we only have stake for NO, then we show, as above, the horizontal radio buttons.

Here are the screenshots;

1. Stake for YES
![Screenshot from 2021-04-20 13-19-52](https://user-images.githubusercontent.com/1193222/115380318-1bb9c000-a1db-11eb-8b35-d48cb4a99c06.png)

2. Stake for YES and NO
![Screenshot from 2021-04-20 13-21-08](https://user-images.githubusercontent.com/1193222/115380485-486dd780-a1db-11eb-9815-e67c9e12ebc6.png)

3. Stakes just for NO
![Screenshot from 2021-04-20 13-20-49](https://user-images.githubusercontent.com/1193222/115380438-3c821580-a1db-11eb-8640-40938d1a8a32.png)

Note that if you have a 100% stake, you need to show the green circle check icon

To have a easier time implementing this, have a look at my #2447 PR since I'm using the same radio buttons to display votes.